libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1098,8 +1100,8 @@ AP_GPS_UBLOX::_save_cfg()
     _last_cfg_sent_time = AP_HAL::millis();
     _num_cfg_save_tries++;
     GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_INFO,
-                                     "GPS: u-blox %d saving config",
-                                     state.instance);
+                                     "GPS %d: u-blox saving config",
+                                     state.instance + 1);
```

Comment:
Good catch! I had gotten the config verification stuff to behave like this but hadn't changed the inside of the drivers/AP_GPS itself to reflect that, that would have been really confusing to a user....